content/en/docs/components/pipelines/concepts/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Concepts"
+description = "Concepts used in Kubeflow Pipelines"
+weight = 30
```

Comment:
NIT: How do you think about changing the weight value to 92? (which is bigger than 90 and smaller than 95) Because the `Concepts` is useful after people installed KFP and started using it.

Reply:
@zijianjoy in my experience, people actually read "concepts" sections before they decide if they want to use a tool, or as a way to understand what a tool does.

For example, see the [argo-workflows docs](https://argoproj.github.io/argo-workflows/core-concepts/), which puts "concepts" second.

Reply:
I see, I don't have a strong opinion against it. Feel free to keep it as is.

Reply:
+1 to Matthew, I think reading concepts earlier makes sense to me